docs/changelog.rst

```diff
@@ -37,6 +37,7 @@ New features:
   ``=~``.
   :bug:`4251`
 * :doc:`/plugins/discogs`: Permit appending style to genre
+* :doc:`plugins/discogs`: Implement item_candidates for matching singletons
```

Comment:
Dot  to finish statement but that's often forgotten by changelog writers, so no worries ;-)

Reply:
Added, thanks! I also fixed the line above.